overlay/etc/nginx/conf.d/20_proxy_cache_path.conf

```diff
@@ -1 +1 @@
-proxy_cache_path /data/cache/cache levels=2:2 keys_zone=generic:CACHE_MEM_SIZE inactive=200d max_size=CACHE_DISK_SIZE loader_files=1000 loader_sleep=50ms loader_threshold=300ms use_temp_path=off;
+proxy_cache_path /data/cache/cache levels=2:2 keys_zone=generic:CACHE_INDEX_SIZE inactive=200d max_size=CACHE_DISK_SIZE loader_files=1000 loader_sleep=50ms loader_threshold=300ms use_temp_path=off;
```

Comment:
Does changing this invalidate CONFIGHASH, i think not as it only uses cache_key not cache_path?

Reply:
I'd say no, since that's compiled in bash land, and not from this. Cache_mem_size is also not reference there.